src/entities/user.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'users' })
```

Comment:
I suggest you removing table name, typeorm will handle it for you changing classname to lowercase and adding plural. The result will be `users`.

```suggestion
@Entity()
```

Reply:
Resolved on 17dd89ff8123681c362f061a4bb75f94f1f18451.